app/controllers/schools/calendars_controller.rb

```diff
@@ -25,12 +26,15 @@ def create
         .where(id: calendar_params[:cohort_ids])
         .pluck(:id)
 
-      @course.calendars.create!(calendar_params)
-
-      authorize(@calendar, policy_class: Schools::CalendarEventPolicy)
+      authorize(@course, policy_class: Schools::CalendarEventPolicy)
```

Comment:
As we are using `CalendarEventPolicy` here as well, considering that the authorization criteria is same. Maybe we can delete `CalendarPolicy` that is unused